src/main/java/com/nike/cerberus/operation/composite/CompositeOperation.java

```diff
@@ -119,16 +96,24 @@ public boolean isRunnable(T command) {
 
             // If the given command is not runnable return false for the whole chain
             //noinspection unchecked
-            if (! operation.isRunnable(chainedCommand)) {
-                return false;
+            boolean isRunnable = operation.isRunnable(chainedCommand);
+            if (! isRunnable) {
+                if (! skipOnNotRunnable()) {
```

Comment:
What is use case for skipOnNotRunnable?  Do we really need this?

Reply:
Not positive but for the compound commands we know of, I think it is just going to fail on following steps and add to the confusion.

Reply:
For example if you temp AWS creds expire half way through standing up a new environment you can just re-run the create environment command and it will skip the commands it has already ran.

It was also nice developing while fixing the bugs from the refactors to be able to run the command over and over and have it skip things it doesn't need to do.

Reply:
Should we call it `--force-continue` or just `--force`?

Reply:
Right now its a method you can overwrite on the CompositeOperation implementation class.

If we want to expose it as a command flag we will need a `CompositeCommandDelagate`.

Reply:
Actually we would need class that all the composite commands extend the same way all the operations extend a composite class